### DISCUSSION TITLE
Welcome to Swar-Chia-Plot-Manager Discussions!

### DISCUSSION THREAD

**Discussion by swar** [swar/Swar-Chia-Plot-Manager#79] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

I've enabled discussion on this repo because I am getting absolutely swamped with tech support questions or questions related to best practices. I figured people can help each other over here or at least look up already asked questions so we can reserve issues with actual bug reports.

**Comment by raid5** [comment]
Awesome, great idea! I've been hesitant to reach out to you on Keybase for this exact reason.

**Comment by LaHood0815** [comment]
great. Can u pls add in the readme if the GUI can be open at the same time as swar chia plot manager runs or must it closed before? Im not sure what is correct. I would like to have the GUI already open to see if coins come in and stuff like that but if that produce errors i wouldnt do that.

merci :)
